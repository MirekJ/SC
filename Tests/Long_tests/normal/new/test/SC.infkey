89ad4f6a-945c-f31d-62c0-75fe1f914532
