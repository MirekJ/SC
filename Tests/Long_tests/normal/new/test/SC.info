<?xml version="1.0" encoding="UTF-8"?>
<job_info>
    <infinity version="INFINITY_INFO_v_2_0" site="{ROBOX:ca923bb5-d4ed-42a3-a709-a4aae64d7931}" server="robox.ncbr.muni.cz"/>
    <basic timedate="1455888826" stimedate="2016-02-19 14:33:46">
        <external>
            <item name="INF_EXTERNAL_NAME_SUFFIX" value=""/>
            <item name="INF_EXTERNAL_VARIABLES" value=""/>
            <item name="INF_EXTERNAL_START_AFTER" value=""/>
        </external>
        <arguments>
            <item name="INF_ARG_DESTINATION" value="long"/>
            <item name="INF_ARG_JOB" value="SC"/>
            <item name="INF_ARG_RESOURCES" value=""/>
            <item name="INF_ARG_SYNC_MODE" value=""/>
        </arguments>
        <jobinput>
            <item name="INF_JOB_PATH" value="/home/xsukenik/git/SC/SC/Tests/Long_tests/normal/new/test"/>
            <item name="INF_JOB_MACHINE" value="lop.ncbr.muni.cz"/>
            <item name="INF_JOB_NAME" value="SC"/>
            <item name="INF_JOB_TYPE" value="generic"/>
            <item name="INF_EXCLUDED_FILES" value=""/>
            <item name="INF_JOB_PROJECT" value=""/>
            <item name="INF_JOB_TITLE" value="SC"/>
            <item name="INF_JOB_KEY" value="89ad4f6a-945c-f31d-62c0-75fe1f914532"/>
        </jobinput>
        <modules>
            <item name="INF_EXPORTED_MODULES" value=""/>
        </modules>
    </basic>
    <specific>
        <resources>
            <item name="INF_UGROUP" value="students"/>
            <item name="INF_UMASK" value="022"/>
            <item name="INF_FS_TYPE" value="consistent"/>
            <item name="INF_ALIAS" value="long"/>
            <item name="INF_ALIAS_RESOURCES" value="props=student"/>
            <item name="INF_ALIAS_SYNC_MODE" value="sync"/>
            <item name="INF_DEFAULT_RESOURCES" value="maxcpuspernode=4"/>
            <item name="INF_QUEUE" value="long"/>
            <item name="INF_NCPU" value="1"/>
            <item name="INF_NGPU" value="0"/>
            <item name="INF_MAX_CPUS_PER_NODE" value="4"/>
            <item name="INF_NNODE" value="1"/>
            <item name="INF_RESOURCES" value="nodes=1:ppn=1:student,props=student,maxcpuspernode=4"/>
            <item name="INF_SURROGATE_MACHINE" value="lop.ncbr.muni.cz"/>
            <item name="INF_SYNC_MODE" value="sync"/>
            <item name="INF_SCRATCH_TYPE" value="local"/>
            <item name="INF_UGROUP_REALM" value=""/>
            <item name="INF_UGROUP_ORIG" value=""/>
        </resources>
    </specific>
    <submit timedate="1455888826" stimedate="2016-02-19 14:33:46">
        <job>
            <item name="INF_JOB_ID" value="2947.robox.ncbr.muni.cz"/>
        </job>
    </submit>
    <start timedate="1455888828" stimedate="2016-02-19 14:33:48">
        <workdir>
            <item name="INF_MAIN_NODE" value="nela.ncbr.muni.cz"/>
            <item name="INF_WORK_DIR" value="/scratch/xsukenik/2947.robox.ncbr.muni.cz/main"/>
        </workdir>
        <nodes>
            <node name="nela.ncbr.muni.cz"/>
        </nodes>
        <gpus/>
    </start>
    <stop timedate="1455916265" stimedate="2016-02-19 22:11:05">
        <result>
            <item name="INF_JOB_EXIT_CODE" value="0"/>
        </result>
    </stop>
</job_info>
