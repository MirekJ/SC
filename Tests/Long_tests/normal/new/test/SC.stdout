Using RAN2 random generator

Patchy Spherocylinders version 3.6
-------------------------------------
Reading options...
 Pressure coupling type:                             1
 Pressure:                                           0.00000000
 Replica exchange pressure:                          0.00000000
 Average volume change attempts per sweep:           0.00000000
 Equilibration sweeps:                               0
 Sweeps between step size adjustments:               0
 Production sweeps:                                  30000000
 Sweeps between statistics samples:                  50000
 Sweeps between statistics reports:                  50000
 Average chain move attempts per sweep:              0.00000000
 Initial maximum displacement:                       0.21200000
 Inititial maximum angular change (degrees):         7.50000000
 Inititial maximum angular cone angle (degrees):     0.00000000
 Inititial maximum geo.box edge change:              0.00000000
 Initial maximum chain displacement:                 0.00000000
 Inititial maximum chain angular change (degrees):   0.00000000
 Temperature in kT/e:                                0.24426000
 Parallel tempering temperature in kT/e:             0.28426000
 Sweeps between replica exchange:                    0
 Sweeps between Grand-Canonical move:                0
 Sweeps between Cluster moves:                       0
 Wang-Landau method:                                 0 0
 Calculate the Wang-Landau method for atom type:     0
 Average type switch attempts per sweep:             0.00000000
 Number of Sweeps per pairlist update:               0
 Random number seed:                                 196150684
 Number of sweeps per writing out cluster info:      50000
 Sweeps between movie frames:                      50000

Reading topology...
Species:
Fields:11 0
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 

Topology read for molecule: A 
particle 1: 	1 
system:  A 164

Topology succesfully read. Generating pair interactions...
Generating GroupLists...

Reading configuration...
Box: (20, 20, 20)
Equilibration of maximum step sizes: 0 sweeps

Initializing energy functions...
Production run:  30000000 sweeps

sweep: 3000000 particles: 164 drift: -9.09495e-13
InteractionEnergy: -295.975, sweeps per hour: 3.93381e+06

sweep: 6000000 particles: 164 drift: 1.81899e-12
InteractionEnergy: -286.7, sweeps per hour: 3.93158e+06

sweep: 9000000 particles: 164 drift: -1.90994e-11
InteractionEnergy: -279.984, sweeps per hour: 3.92443e+06

sweep: 12000000 particles: 164 drift: -2.63753e-11
InteractionEnergy: -309.588, sweeps per hour: 3.93219e+06

sweep: 15000000 particles: 164 drift: -1.63709e-11
InteractionEnergy: -281.661, sweeps per hour: 3.93182e+06

sweep: 18000000 particles: 164 drift: -1.63709e-11
InteractionEnergy: -296.984, sweeps per hour: 3.93466e+06

sweep: 21000000 particles: 164 drift: -2.09184e-11
InteractionEnergy: -314.296, sweeps per hour: 3.93553e+06

sweep: 24000000 particles: 164 drift: -1.45519e-11
InteractionEnergy: -288.107, sweeps per hour: 3.93509e+06

sweep: 27000000 particles: 164 drift: 4.54747e-12
InteractionEnergy: -291.716, sweeps per hour: 3.93379e+06

sweep: 30000000 particles: 164 drift: 1.45519e-11
InteractionEnergy: -299.934, sweeps per hour: 3.94115e+06

Energy drift: 1.45519e-11 
Starting energy+pv: -6335.54617004 
Starting energy: -301.40108453 
Ending energy: -300.37725184 


******************************************************************************
*                               Moves Statistics                             *
******************************************************************************
Move                          Acc (%)   Rej (%)   Steps     
System:
A 164

Simulation time:2.7e+04 s
PairList generation time:0 %

DONE
Deallocating Conf...
Deallocating Sim...
Deallocating Topo...
